include/Metabox/Posts/CustomTaxonomy.php

```diff
@@ -26,7 +26,6 @@ public function render() {
 					<td scope="row" colspan="2">
 						<?php _e( 'Select the post type to show its custom taxonomy.', 'bulk-move' ); ?>
 					</td>
-					<td scope="row">
```

Comment:
Any specific reason for removing these `td`s?

Reply:
Its unwanted td and its not properly closed

Reply:
👍